docker/worldping-api/worldping-api.ini

```diff
@@ -20,7 +20,7 @@ cert_key =
 admin_key = changeme
 
 [raintank]
-graphite_url = http://graphite-api:8888/
+graphite_url = http://metrictank:6063/
```

Comment:
Well, the graphite server is used for both serving requests for worldping-app and also for alerting queries.
I had this pointed to MT for for testing, but your right it should point to to graphite-metrictank